src/Poser.php

```diff
@@ -10,8 +10,6 @@ class Poser
 
     /**
      * Constructor.
```

Comment:
Maybe we'll drop all the docblock? Since it's useless

Reply:
right! in the next PR! ;)